lib/logstash/inputs/beats.rb

```diff
@@ -175,6 +177,7 @@ def register
   def create_server
     server = org.logstash.beats.Server.new(@host, @port, @client_inactivity_timeout)
     if @ssl
+      check_no_exec_mount!
```

Comment:
woa,  that make the code a lot simplier I wasn't aware of that method, I guess I've looked at the wrong place.